Dockerfile.android

```diff
@@ -0,0 +1,38 @@
+
+FROM python:3.11.3-alpine
```

Comment:
I'm not a big Python person, but should this be 3.11.x or 3.12?
Per python.org:
> Latest: [Python 3.12.1](https://www.python.org/downloads/release/python-3121/)

Reply:
I'll have to do this later as there are some changes needed upstream in the android repo